src/server/controllers/signInControllers.ts

```diff
@@ -397,17 +397,22 @@ export const oktaIdxApiSignInController = async ({
 	req: Request;
 	res: ResponseWithRequestState;
 }) => {
-	// get the email and password from the request body
+	// get the email and password from the request body if using passwords
+	// or the "passcode" parameter is a hidden input, which is to determine if the
+	// user is signing in with a passcode and not an actual passcode value
```

Comment:
> // user is signing in with a passcode and not an actual passcode value

What is the difference between 'passcode` and `passcode value` ?

Reply:
It's not a "code" like "123456", but a key/value set to passcode coming from: https://github.com/guardian/gateway/blob/f28db8f39c863b4831140bff873f7d150eed3fdf/src/client/pages/SignIn.tsx#L210